### PR TITLE
[#1034] Revert code

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses issue https://github.com/episphere/connect/issues/1034

ChangeLog

- Moved `const batch = db.batch();` inside the condition.

### CODE REVIEW

**Comment by jhflorey** [comment]
@JoeArmani @we-ai @anthonypetersen  Regarding this comment https://github.com/episphere/connectFaas/pull/608#issuecomment-2192446030

- `const batch = db.batch();` was inside the loop before and the PR https://github.com/episphere/connectFaas/pull/608 just only moves it inside the right condition.
- This batch is not used to update multiple participants. It is used to update data of each participant (Remove data + Update 2 participant statuses)

**Comment by anthonypetersen** [comment]
Is there a point in creating batches if we are only updating one participant at a time?

**Comment by we-ai** [comment]
`batch` is meant for batched updates. Using batch to update a single doc is not necessary.

**Comment by jhflorey** [comment]
> `batch` is meant for batched updates. Using batch to update a single doc is not necessary.

Do i need to update to normal way.

**Comment by anthonypetersen** [comment]
I agree with Warren - if you are only updating one document, using `batch` feels unnecessary. You can just use an object to maintain what is going to be updated for the participant and then update the document at the end of your logic.

**Comment by JoeArmani** [comment]
I agree with Tony's and Warren's assessments on this.

**Comment by jhflorey** [comment]
@anthonypetersen @JoeArmani @we-ai just updated and tested with participant (Connect_ID 7009372190)

File: utils/firestore.js [hunk 1]
```diff
@@ -447,12 +447,11 @@ const removeParticipantsDataDestruction = async () => {
             .where(dataHasBeenDestroyed, "!=", fieldMapping.yes)
             .get();
 
-        const batch = db.batch();
-
         // Check each participant if they are already registered or more than 60 days from the date of their request
         // then the system will delete their data except the stub records and update the dataHasBeenDestroyed flag to yes.
         for (const doc of currSnapshot.docs) {
             const participant = doc.data();
+            const participantId = doc.id;
```
Comment (anthonypetersen) [inline utils/firestore.js:454]: NIT - the document ID doesn't represent the ID of the participant

**Review by JoeArmani — APPROVED** [review]

File: utils/firestore.js (hunk 1 above)
Comment (we-ai) [reply utils/firestore.js:454]: `docId` is a better name.
This line is not needed, since `doc.id` can be directly used and is clearer.

**Pull request merged**